Data Structures/Array/unique_element.java

```diff
@@ -0,0 +1,73 @@
+		Scanner scn = new Scanner(System.in);
```

Comment:
@Sandip75, It looks like the code above this line is missing.

Reply:
Yes I removed  it 
According @arnav-mandal1234 told me

Reply:
Yes, but he told only to remove the package part.

Reply:
Should I add ?

Reply:
Add the part below the *package* statement, otherwise the code won't compile without a class definition.

Reply:
okay